File: src/main/java/org/chousik/collection/Color.java
package org.chousik.collection;

import java.util.Arrays;
import java.util.LinkedList;
import java.util.stream.Collectors;

/**
 * Класс перечисление цветов
 */
public enum Color {
    GREEN,
    YELLOW,
    ORANGE,
    BROWN;

    /**
     * @return возвращает все значения перечисления
     */
    public static String getValue() {
        return Arrays.stream(Color.values())
                .map(Color::name)
                .collect(Collectors.joining(", "));
    }
}

File: src/main/kotlin/collection/Color.kt
package org.chousik.collection

import java.util.*
import java.util.stream.Collectors


enum class Color {
    GREEN,
    YELLOW,
    ORANGE,
    BROWN;

    companion object {
        val value: String
            get() = Arrays.stream(entries.toTypedArray())
                .map({ obj: Color -> obj.name })
                .collect(Collectors.joining(", "))
    }
}